File: C2108G2/MVC/ECommerceApp/Models/Book.cs
using Microsoft.AspNetCore.Authentication;

namespace ECommerceApp.Models
{
    public class Book
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public float Price { get; set; }
        public string Description { get; set; }
        
    }   
}

File: C1908GDoiCan/IASF/11-09-2021/springde01/src/main/java/com/exam/springde01/models/Book.java
package com.exam.springde01.models;
import javax.persistence.*;
import javax.validation.constraints.Min;

@Entity
@Table(name = "Books")
public class Book {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO) //auto-increment
    private Long bookId;
    private String title;
    @Min(value=0)
    private Float price;
    private Long categoryId;
}
